### DISCUSSION TITLE
Screen Recorder by default does not save the screenshot to a file

### DISCUSSION THREAD

**Discussion by igatjens** [linuxdeepin/developer-center#2959] [Features Request & Ideas | 特性请求 & 头脑风暴]
Screen Recorder by default saves the capture only to the clipboard.

In previous versions, by default, Screen Recorder has always saved screenshots on the desktop. But version 5.8.0.40-1 by default only saves the capture to the clipboard.

It is more convenient and is what old users expect, that the default capture is saved to the desktop and the clipboard at the same time.

It is uncomfortable that the first capture after installing Deepin, is not saved on the desktop. You have to redo the capture and manually configure where it will be saved. It is a small hiccup in the user experience.

I also want to tell you that I love that the capture is kept on the desktop and at the same time, it is very useful especially when sending screenshots to help other users on Telegram.

```
apt list deepin-screen-recorder
Listando... Hechodeepin-screen-recorder/desconocido,now 5.8.0.40-1 amd64 [instalado]
deepin-screen-recorder/desconocido 5.8.0.40-1 i386
```

![20200920_082207](https://user-images.githubusercontent.com/66392244/93714205-5745bc00-fb1e-11ea-8519-cc915608dfbe.jpg)

Escritorio → Desktop
Imágenes → Image
Carpeta → Folder

**Comment by meiyixiang** [comment]
Now added many personalization Settings you can customize save in clipboard, file directory, desktop etc different ways, please upgrade to 20.5
